Configuration/PyReleaseValidation/README.md

```diff
@@ -61,6 +61,8 @@ The offsets currently in use are:
 * 0.7: trackingMkFit modifier
 * 0.701: DisplacedRegionalStep tracking iteration for Run-3
 * 0.702: trackingMkFit modifier for Phase-2 (initialStep only)
+* 0.703: LST tracking, initialStep+HighPtTripletStep only, on CPU
+* 0.704: LST tracking, initialStep+HighPtTripletStep only, on GPU
```

Comment:
Another thing: the description says "on GPU", but the implementation is "on NVIDIA GPU".

Would you rather add a different workflow for AMD GPUs (and, maybe in the future, Intel GPUs) ?
Or change the implementation to be "on any GPU" ?
That is
```diff
-    '--accelerators' : 'gpu-nvidia'
+    '--accelerators' : 'gpu-*'
```